kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/manager/ResourceManager.java

```diff
@@ -110,6 +130,9 @@ private final <T extends HasMetadata> void createResource(boolean waitReady, T..
 
             assert type != null;
             type.create(resource);
+
+            storedResources.computeIfAbsent(getTestContext().getDisplayName(), k -> new Stack<>())
```

Comment:
`Stack` is a historic class.  The javadoc suggests it is not used.  `ConcurrentLinkedDeque` might be a better choice.